### PR TITLE
issue-136: re-export ./util entities

### PR DESCRIPTION
**Pull request by syavorsky**
@brettz9 would this suffice? I am not familiar with behavior described in #136, so just blindly taking your advice

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import alignTransform from './transforms/align';
 import indentTransform from './transforms/indent';
 import crlfTransform from './transforms/crlf';
 import { flow as flowTransform } from './transforms/index';
+import { rewireSpecs, rewireSource, seedBlock, seedTokens } from './util';
```
Comment (brettz9) [inline src/index.ts:11]: Here's a more succinct alternative, assuming TS supports it:

```suggestion
export { rewireSpecs, rewireSource, seedBlock, seedTokens } from './util';
```

Then you can get rid of the other export.

File: src/index.ts [hunk 2]
```diff
@@ -31,3 +32,5 @@ export const tokenizers = {
   name: nameTokenizer,
   description: descriptionTokenizer,
 };
+
+export const util = { rewireSpecs, rewireSource, seedBlock, seedTokens };
```
Comment (brettz9) [inline src/index.ts:36]:
```suggestion
```

File: src/index.ts [hunk 3]
```diff
@@ -31,3 +32,5 @@ export const tokenizers = {
   name: nameTokenizer,
   description: descriptionTokenizer,
 };
+
```
Comment (brettz9) [inline src/index.ts:35]:
```suggestion
```

**Comment by brettz9** [comment]
Yeah, that would be great...

File: src/index.ts (hunk 1 above)
Comment (syavorsky) [reply src/index.ts:11]: if I read it right, all entities will be exported as top-level names which I was explicitly trying to avoid

Comment (brettz9) [reply src/index.ts:11]: Oh yes, sorry, I see why you were doing that. Right, you can just leave it all as is then.

Comment (syavorsky) [reply src/index.ts:11]: cool, merging then

File: src/index.ts (hunk 3 above)
Comment (syavorsky) [reply src/index.ts:35]: I believe you were the one adding [insert_final_newline = true](https://github.com/syavorsky/comment-parser/blob/master/.editorconfig#L9) :)

Comment (brettz9) [reply src/index.ts:35]: :-) Another oops... Here I thought I had been deleting the extra line break from the export! Anyways, glad it wasn't needed.

**Pull request merged**